src/FastBoard.cpp

```diff
@@ -50,8 +50,8 @@ int FastBoard::get_boardsize(void) const {
 }
 
 int FastBoard::get_vertex(int x, int y) const {
-    assert(x >= 0 && x < MAXBOARDSIZE);
-    assert(y >= 0 && y < MAXBOARDSIZE);
+    assert(x >= 0 && x < BOARD_SIZE);
+    assert(y >= 0 && y < BOARD_SIZE);
     assert(x >= 0 && x < m_boardsize);
```

Comment:
long term we can remove m_boardsize.

Reply:
That'd mean removing get_boardsize() and replacing every call with BOARD_SIZE. I'll do this tonight.